lib/modules/datasource/gitlab-tags/util.ts

```diff
@@ -7,6 +8,11 @@ export function getDepHost(registryUrl: string = defaultRegistryUrl): string {
   return registryUrl.replace(regEx(/\/api\/v4$/), '');
 }
 
+export function getDefaultRegistryUrl(): string[] {
+  const { platform, endpoint } = GlobalConfig.get();
+  return platform === 'gitlab' && endpoint ? [endpoint] : [defaultRegistryUrl];
+}
```

Comment:
I thought that our managers all extract with registryUrls specified today.

How to test/coverage this new function though if no implementations?

Reply:
> This will break users when they are on a self-hosted gitlab instance and referencing dependencies from public hosted `gitlab.com`

So should we make it platform independent?

Reply:
Not sure what we need to do here.